src/bots/filler.ts

```diff
@@ -897,6 +900,72 @@ export class FillerBot implements Bot {
 		return true;
 	}
 
+	protected selectSubsetOfMakers(nodeToFill: NodeToFill) {
+		let makerNodes = nodeToFill.makerNodes;
+		if (makerNodes.length >= MAX_MAKERS_PER_FILL) {
+			// Calculate the values for each item and sum them up
+			const values: BN[] = [];
+			const totalValue = ZERO;
+			for (const node of makerNodes) {
+				if (node.order) {
+					const value = node.order.baseAssetAmount.sub(
+						node.order.baseAssetAmountFilled
+					);
+					values.push(value);
+					totalValue.add(value);
```

Comment:
you're summing up the base amts here, so actually biasing for large orders?

if you want to bias towards the front of book don't you need to sort by price?